src/core.py

```diff
@@ -191,3 +191,39 @@ class Fiat(Enum):
     ZAR = "South Africa Rand"
     ZMW = "Zambia Kwacha"
     ZWD = "Zimbabwe Dollar"
+
```

Comment:
I created these mappings using a simple python script which fetches the currently supported AssetPairs JSON and checks which pairs need "special attention" as per the "kraken_pair_map".

[Kraken-AssetPairs-Parse.py.txt](https://github.com/provinzio/CoinTaxman/files/6028235/Kraken-AssetPairs-Parse.py.txt)